src/aiotk/_sched.py

```diff
@@ -22,24 +25,29 @@ class PeriodicTask:
 
     """
 
-    def __init__(self, func, interval, loop=None):
+    def __init__(self, func: Callable,
+                 interval: float,
+                 loop: Optional[AbstractEventLoop]=None) -> None:
         self._loop = loop or asyncio.get_event_loop()
         self._func = func
         self._ival = interval
-        self._task = None
+        self._task = None  # type: Optional[Task]
 
+    # NOTE: cannot declare type for return value here because the class
+    #       definition is not completed...
     async def __aenter__(self):
```

Comment:
**Note to self**: report this feedback to `mypy` devs to ensure they are aware of this limitation in Python.